js/ui/number_box/number_box.mask.js

```diff
@@ -243,6 +243,11 @@ const NumberBoxMask = NumberBoxBase.inherit({
             if(this._parsedValue < 0 || 1 / this._parsedValue === -Infinity) {
                 this._revertSign(e);
                 this._setTextByParsedValue();
+
+                if(this.option('valueChangeEvent') === 'input') {
```

Comment:
The 'input' event will not trigger for cases when we mix `valueChangeEvent` events.
For example this case:
`valueChangeEvent: 'input focusout'`. It should be work too.

Reply:
thanks, i missed that it can accept several events